src/Compilers/Core/Portable/Serialization/ObjectData.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis;
+
+namespace Roslyn.Utilities
+{
+    internal class ObjectData
```

Comment:
That's true.. but what I really meant was I needed the immutable object array to be a reference type (with reference identity) so I can associate other data to with it.

---
In reply to: [85239590](https://github.com/dotnet/roslyn/pull/14671#discussion_r85239590) [](ancestors = 85239590)